hosters/hetzner-dedicated/hetzner-dedicated-wipe-and-install-nixos.sh

```diff
@@ -78,21 +140,26 @@ parted --script /dev/sdb mklabel gpt
 #   ... part-type is one of 'primary', 'extended' or 'logical', and may be specified only with 'msdos' or 'dvh' partition tables.
 #   A name must be specified for a 'gpt' partition table.
 # GPT partition names are limited to 36 UTF-16 chars, see https://en.wikipedia.org/wiki/GUID_Partition_Table#Partition_entries_(LBA_2-33).
-parted --script --align optimal /dev/sda -- mklabel gpt mkpart 'BIOS-boot-partition' 1MB 2MB set 1 bios_grub on mkpart 'data-partition' 2MB '100%'
-parted --script --align optimal /dev/sdb -- mklabel gpt mkpart 'BIOS-boot-partition' 1MB 2MB set 1 bios_grub on mkpart 'data-partition' 2MB '100%'
+for disk in "${DISKS[@]}"; do
+  parted --script --align optimal "/dev/$disk" -- \
+         mklabel gpt \
+         mkpart 'BIOS-boot-partition' 1MB 2MB \
+         set 1 bios_grub on \
+         mkpart 'data-partition' 2MB '100%'
+done
 
 # Relaod partitions
 partprobe
 
-# Wait for all devices to exist
-udevadm settle --timeout=5 --exit-if-exists=/dev/sda1
-udevadm settle --timeout=5 --exit-if-exists=/dev/sda2
-udevadm settle --timeout=5 --exit-if-exists=/dev/sdb1
-udevadm settle --timeout=5 --exit-if-exists=/dev/sdb2
+for disk in "${DISKS[@]}"; do
+  # Wait for all devices to exist
+  udevadm settle --timeout=5 --exit-if-exists="/dev/${disk}1"
```

Comment:
Will this work? `/dev/sda` would have `/dev/sda1`, but `/dev/nvme0n1` would have `/dev/nvme0n1p1`

Reply:
Oh you're right, I'll have to fix that.

Reply:
Should be fixed, but untested.